sdk/storage/azure-storage-common/inc/azure/storage/common/internal/constants.hpp

```diff
@@ -8,11 +8,14 @@ namespace Azure { namespace Storage { namespace _internal {
   constexpr static const char* DatalakeServicePackageName = "storage-files-datalake";
   constexpr static const char* FileServicePackageName = "storage-files-shares";
   constexpr static const char* QueueServicePackageName = "storage-queues";
+  constexpr static const char* TablesServicePackageName = "storage-tables";
   constexpr static const char* HttpQuerySnapshot = "snapshot";
   constexpr static const char* HttpQueryVersionId = "versionid";
   constexpr static const char* HttpQueryTimeout = "timeout";
   constexpr static const char* StorageScope = "https://storage.azure.com/.default";
   constexpr static const char* StorageDefaultAudience = "https://storage.azure.com";
+  constexpr static const char* TablesManagementScope = "https://management.azure.com/.default";
+  constexpr static const char* TablesManagementPublicEndpoint = "https://management.azure.com";
```

Comment:
I believe storage management plane APIs should be in a separate package. For example, in .Net https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/storage/Microsoft.Azure.Management.Storage.

It'd be best to not mix them.